client/utils/account-fees.tsx

```diff
@@ -85,19 +85,36 @@ export const getCurrentBaseFee = (
 		: accountFees.base;
 };
 
+export const getTooltipAndPillBaseFee = (
```

Comment:
I think your analysis is accurate here: we need `percentage_rate` and `fixed_rate` and falling back to the base fee structure seems a good idea to me.

But I am not convinced about adding this new function. I found this part of the code a bit hard to follow, even before this PR, and adding this new function was not very helpful in understanding what is going on.

Reply:
Marking this as resolved for now, since the function `getTooltipAndPillBaseFee` has been removed. We are discussing separately about the new `getCurrentFeeWithBaseFallback` [function here](https://github.com/Automattic/woocommerce-payments/pull/5378#discussion_r1068383656). Thanks